src/dbg/taskthread.cpp

```diff
@@ -0,0 +1,95 @@
+#include <Windows.h>
```

Comment:
There is no need to move it, just delete it. Global includes all windows
headers. I will update visual studio 2013 to the latest version.
On Tue, 5 Jul 2016 at 23:20, jdavidberger <notifications@github.com> wrote:

> In src/dbg/taskthread.cpp
> <https://github.com/x64dbg/x64dbg/pull/821#discussion_r69640294>:
>
> > @@ -0,0 +1,95 @@
> > +#include <Windows.h>
>
> Even moving that around I get this kind of thing:
> https://gist.github.com/jdavidberger/d6b93ffed5195a7ad6139f5d907fc278 in
> other compilation units. cl is giving a version as Version 18.00.40629 for
> x86 and VS2013 version 12.0.40629.00 update 5 for what its worth.
>
> This does bring up though that apparently InitializeConditionVariable
> isn't available on XP. I'll reimplement TaskThread with a semaphore and fix
> the header order.
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/x64dbg/x64dbg/pull/821/files/df25097e92d11eb5b60b11d3cb1144f39ef3ff45#r69640294>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACWCmX0XlOnNM79luFNqZnZ01bxmOKaBks5qSsq7gaJpZM4JE7hu>
> .
>